src/course-unit/__mocks__/clipboardResponse.js

```diff
@@ -0,0 +1,9 @@
+module.exports = {
+  locator: 'block-v1:edX+DemoX+Demo_Course+type@vertical+block@vertical_0270f6de40fc',
+  courseKey: 'course-v1:edX+L153+3T2023',
+  static_file_notices: {
```

Comment:
[question] Do we need to use here snake_case?

Reply:
Nope, changed